### PR TITLE
Added sql-practice.com as an example.

### PR DESCRIPTION
**Pull request by 000744210**
sql-practice is a practical example of implementing sql.js on a website.

I am not a professional at using git so I'm sorry if I made a mistake when submitting this.

I noticed it gives a "unknown date" for the date added field. I don't know if there's something I had to do to fix this or it will just work when its hosted.

### CODE REVIEW

File: showcase/sql-practice.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+website: https://www.sql-practice.com
+description: A practical example of sql.js being implemented on a site.
+keywords: language javascript vue sql
+---
+
+Sql-practice is a site designed to help people practice and learn sql by solving sql query problems.
+
+![sql-practice screenshot](https://i.imgur.com/YOCFVxd.png)
+
+sql-practice was written to use sql.js in a practical use. Implementing this site prior to sql.js would have required a lot of server cost but now it can be done on a static site.
```
Comment (torch2424) [inline showcase/sql-practice.md:13]: Could you mention here that: ["sql.js uses emscripten to compile SQLite to webassembly "](https://sql.js.org/#/). 😄

**Review by torch2424 — CHANGES_REQUESTED** [review]
> sql-practice is a practical example of implementing sql.js on a website.

Ah awesome! Thank you very much for the contribution! 😄 

> I am not a professional at using git so I'm sorry if I made a mistake when submitting this.

No worries at all! You did a great job here, and everything looks correct (In terms of opening a PR)

> I noticed it gives a "unknown date" for the date added field. I don't know if there's something I had to do to fix this or it will just work when its hosted.

That's okay! 😄  We generate the date for the examples on build: https://github.com/torch2424/made-with-webassembly/blob/master/showcase-build.js#L54 . So after this is merged, it'll show up correctly! 

---

All in all, I just have one request and then I'm happy to merge this. Thank you! 😄

File: showcase/sql-practice.md (hunk 1 above)
Comment (000744210) [reply showcase/sql-practice.md:13]: I believe I made the change. 

Thank you for maintaining the site by the way. It helped me get inspiration for other wasm-based sites.

**Comment by torch2424** [comment]
I was out last week from work, so I took a quick Github break as well! This now looks great, approving and merging! Thank you for the contribution @000744210 !

**Review by torch2424 — APPROVED** [review]

**Pull request merged**